nixos/modules/services/misc/gitolite.nix

```diff
@@ -71,7 +95,6 @@ in
     systemd.services."gitolite-init" = {
       description = "Gitolite initialization";
       wantedBy    = [ "multi-user.target" ];
-      unitConfig.RequiresMountsFor = cfg.dataDir;
```

Comment:
Removing this line looks like a bad conflict resolution.

Reply:
Right.